### PR TITLE
Proper UI elements showing

### PR DESCRIPTION
**Pull request by Georgehe4**
@devoncarew 
Need to add a few styling conditions to make it look smoother but base functionality is there.

### CODE REVIEW

**Comment by devoncarew** [comment]
Cool! Can you add a screenshot to the PR, and run `lib/documentation.dart` through the formatter? I think there are some indentation issues.

**Comment by Georgehe4** [comment]
screen shot 2015-06-25 at 9 24 11 am
screen shot 2015-06-25 at 9 24 34 am
Screenshots @devoncarew

File: lib/embed/embed.dart [hunk 1]
```diff
@@ -479,10 +482,19 @@ class PlaygroundMobile {
...
-  void _showOuput(String message, {bool error: false}) {
+    _output.add(new DivElement()
+      ..classes.addAll([
+        'consoleTitle',
+        'center-center',
+        'height-max',
+        'layout',
+        'horizontal'
+      ])
+      ..innerHtml = "Console Output Area");
```
Comment (devoncarew) [inline lib/embed/embed.dart]: `Console output`?

File: web/embed_components.html [hunk 2]
```diff
@@ -221,6 +221,8 @@
       <div class="layout horizontal flex-4">
         <paper-tabs selected="0" class="tabs flex">
           <span class="flex"></span>
+          <div class="center-center horizontal layout">HTML Output</div>
```
Comment (devoncarew) [inline web/embed_components.html]: I don't think we need the title here.

**Comment by devoncarew** [comment]
lgtm

I think we can iterate on the look of the watermark text a bit, but that can be in a future PR.

**Comment by Georgehe4** [comment]
#535 
Merging.

**Pull request merged**